.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!--
+Thank you for reporting a bug in Boa! This will make us improve the engine. But first, fill the following template so that we better understand what's happening. Feel free to add or remove sections as you feel appropriate.
+-->
+
+**Describe the bug**
+A clear and concise description of what the bug is. E.g:
+
+The variable statement is not working as expected, it always adds 10 when assigning a number to a variable"
```

Comment:
So, I thought about this, but I was unsure on what to do. If we put everything in comments, it would look strange, but on the other hand, these are just examples.

You think it would be better as a comment?

Reply:
> So, I thought about this, but I was unsure on what to do. If we put everything in comments, it would look strange, but on the other hand, these are just examples.
> 
> You think it would be better as a comment?

Yes. Making the comments emphasizes that it's only an example.